### ISSUE TITLE
Application Steals Focus From Other Windows

### ISSUE THREAD

**Issue by unicornsoftwareinc** [MichaelDimmitt/Aerial_Desktop#36]
When I leave this desktop background running for long periods of time, it seems to start stealing focus away from other applications that I am currently using. It also seems like the frequency of focus stealing increases over time.

It was stealing focus from whatever my active application was literally every 5 - 10 seconds before I turned the desktop background off.

**Comment by jspinella** [comment]
Does the background scene change when focus is stolen? I assume not for the every 5-10 seconds scenario, but before that, when it's occasional? Does focus go to Finder or Terminal?

**Comment by unicornsoftwareinc** [comment]
No, it doesn't change the scene. And it always just switched focus to Finder

**Comment by jspinella** [comment]
Gotcha. What's your Mac and version of macOS? How long does it usually take for the focus-stealing to start?

**Comment by MichaelDimmitt** [comment]
Hi @hugeprestigellc, interesting issue. 
It will be very useful if I can recreate the problem, 

Taking a screenshot may be tough for capturing the issue.

Instead of taking a screenshot. A photo of the screen from your phone or external camera when the problem is actually happening could be very helpful.

Then you could download the photo to your computer. 
Photo files can be attached to the issue page by copy and paste into a comment.
Or dragging directly into a new comment.

**Comment by unicornsoftwareinc** [comment]
Just re-confirmed this issue again today. It seems like when I have the desktop background going for more than 24 hours that the focus loss starts occurring.

I'm not really sure what I would take a photo of here unfortunately. It's just the app that has focus seems to lose it and the focus switches to Finder. As soon as I turn off the desktop background the focus loss issue goes away.

**Comment by MichaelDimmitt** [comment]
@hugeprestigellc, I can write a condition in the monitor launch agent to kill and restart the background changer. This is currently second on my to do, list. Currently I am working to get the application working for all versions of Mac Computers. And introduce the menubar implementation as an option.

@hugeprestigellc, for documentation, A photo of the screen from your phone or external camera when the problem is actually happening could be very helpful.

In the meantime, this week I plan to make a branch and write the restart logic. 
Then you will be able to check out to that branch.

If you are not a coder I will also add release. a packaged app file in this issues page.
That you will be able to download and run.

**Comment by unicornsoftwareinc** [comment]
Thanks. I'll try to get a recording of the issue next time I have the background running. 

Also, I'm running macOS Sierra 10.12.6 on a MacBook Pro (15-inch, 2016) if that helps at all.
